### PR TITLE
Add plan review for MP 6.0

### PR DESCRIPTION
**Pull request by Emily-Jiang**

### CODE REVIEW

File: microprofile-6.0/README.adoc [hunk 1]
```diff
@@ -4,17 +4,31 @@ date: 2022-09-20
...
+* Jakarta EE 10 Core Profile*
+* Config 3.0.2
+* Fault Tolerance 4.0.1
+* Health 4.0.1
+* JWT RBAC 2.1*
+* Metrics 5.0*
+* Open API 3.1*
+* Rest Client 3.0.1
+* Telemetry 1.0*
+pass:[*] New or Updated
```
Comment (Emily-Jiang) [inline microprofile-6.0/README.adoc:18]: please add a new line before line 18 as it does not render well.

File: microprofile-6.0/README.adoc [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+Config, Fault Tolerance, Health and Rest Client will perform a service release to address CVE-2021-26291.
+
+Jakarta EE 10 Core Profile includes the following specifications, which must be implemented by MicroProfile 6.0 implementations:
+
+Jakarta Annotations 2.1
```
Comment (Emily-Jiang) [inline microprofile-6.0/README.adoc:24]:
```suggestion
* Jakarta Annotations 2.1
```

Comment (Emily-Jiang) [reply microprofile-6.0/README.adoc:24]: please do the same for the following lines.

File: microprofile-6.0/README.adoc [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+* Telemetry 1.0*
+
+pass:[*] New or Updated
+
+Config, Fault Tolerance, Health and Rest Client will perform a service release to address CVE-2021-26291.
+
+Jakarta EE 10 Core Profile includes the following specifications, which must be implemented by MicroProfile 6.0 implementations:
+
+Jakarta Annotations 2.1
+Jakarta Contexts and Dependency Injection (Lite Section) 4.0**
```
Comment (Emily-Jiang) [inline microprofile-6.0/README.adoc:26]:
```suggestion
* Jakarta Contexts and Dependency Injection (Lite Section) 4.0**
```

File: microprofile-6.0/README.adoc [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+Jakarta Dependency Injection 2.0
```
Comment (Emily-Jiang) [inline microprofile-6.0/README.adoc:27]:
```suggestion
* Jakarta Dependency Injection 2.0
```

File: microprofile-6.0/README.adoc [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+Jakarta Interceptors 2.1
```
Comment (Emily-Jiang) [inline microprofile-6.0/README.adoc:28]:
```suggestion
* Jakarta Interceptors 2.1
```

File: microprofile-6.0/README.adoc [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+Jakarta JSON Binding 3.0
```
Comment (Emily-Jiang) [inline microprofile-6.0/README.adoc:29]:
```suggestion
* Jakarta JSON Binding 3.0
```

File: microprofile-6.0/README.adoc [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+Jakarta JSON Processing 2.1
```
Comment (Emily-Jiang) [inline microprofile-6.0/README.adoc:30]:
```suggestion
* Jakarta JSON Processing 2.1
```

File: microprofile-6.0/README.adoc [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+Jakarta RESTful Web Services 3.1
```
Comment (Emily-Jiang) [inline microprofile-6.0/README.adoc:31]:
```suggestion
* Jakarta RESTful Web Services 3.1
```

**Comment by Emily-Jiang** [comment]
@jclingan you fixed my first comment. The following is still not rendered well.
image
You can either accept my suggestion or do another commit to fix all of the lines.

**Comment by JanWesterkamp-iJUG** [comment]
@Emily-Jiang, @jclingan (and @ederks85), I found a few issues:

We will have a microprofile-parent major update, that will include a major release migration for Jakarta EE (9.1 to 10) including the CDI Lite 4 major release (and other things too).
We should not use patch or service releases to reflect this!
Besides issues with semver, if we woud do it like that and we need to fix one of the existing MP component specs, we would have to add another digit to the version, i.e. MP Config 3.0.1.1 - I think this is not a good idea at all...

Additionally, some MP component specs are not named correctly:

- Open API -> OpenAPI
- JWT RBAC -> JWT ?

Sometimes this is a difficult task, as component specs themself do not use consistent naming (i.e. [microprofile-jwt-auth issue 303](https://github.com/eclipse/microprofile-jwt-auth/issues/303) or MP OpenAPI has an inconsistent naming regarding the GitHub repository name with eclipse/microprofile-open-api and on the website too).
If we do not fix these issues, we keep the technical debt and have to pay the interest rate again and again...

Being short in the naming can be a good strategy to prevent issues like this, i.e. MicroProfile Telemetry covers MicroProfile Telemetry Tracing (and that will be included) ;-)

To be exact, it might be useful in the future to use the official full name of the component spec to prevent confusion i.e. when there is a component spec of the same name in Jakarta like Config - using MicroProfile Config and Jakarta Config would help to separate them.

Some of this findings are old or repeated ones (from my MP 5.0 Release Review), that are not addessed yet. I think we should do that and in case this Plan Review passes (formally it looks like now), we can still do a new iteration to address my concerns without delaying the work on the release in general, I think.

As conclusion, I have to vote -1 now, but hoping to get this fixed soon! :-)

**Comment by edbratt** [comment]
Sorry for being a dunderhead, where is this page intended to display -- at microprofile.io, somewhere?

**Comment by jclingan** [comment]
@edbratt  This will be displayed in the [release record](https://projects.eclipse.org/projects/technology.microprofile/releases/6.0) without the asciidoc formatting. It is unclear how it will be referred to from microprofile.io. We are still discussing individual spec pages. In the meantime, it is in asciidoc format here.

@JanWesterkamp-iJUG We should figure out the proper name. We've called it [JWT Auth](https://github.com/eclipse/microprofile-jwt-auth/releases/tag/2.0) for a long time. However, the spec text says [Interoperable JWT RBAC](https://download.eclipse.org/microprofile/microprofile-jwt-auth-2.0/microprofile-jwt-auth-spec-2.0.html). You are correct that this needs to be addressed. I'll fix Open API -> OpenAPI

**Pull request merged**